raiden/tests/utils/factories.py

```diff
@@ -327,39 +372,52 @@ def make_signed_balance_proof(
 
     signature = eth_sign(privkey=private_key, data=data_to_sign)
 
-    signed_balance_proof = BalanceProofSignedState(
-        nonce,
-        transferred_amount,
-        locked_amount,
-        locksroot,
-        token_network_address,
-        channel_identifier,
-        extra_hash,
-        signature,
-        sender_address,
-        UNIT_CHAIN_ID,
+    return BalanceProofSignedState(
+        nonce=nonce,
+        transferred_amount=transferred_amount,
+        locked_amount=locked_amount,
+        locksroot=locksroot,
+        token_network_identifier=token_network_address,
+        channel_identifier=channel_identifier,
+        message_hash=extra_hash,
+        signature=signature,
+        sender=sender_address,
+        chain_id=UNIT_CHAIN_ID,
     )
 
-    return signed_balance_proof
-
 
 def make_signed_transfer_for(
-        channel_state,
-        amount,
-        initiator,
-        target,
-        expiration,
-        secret,
-        identifier=1,
-        nonce=1,
-        transferred_amount=0,
-        locked_amount=None,
-        pkey=UNIT_TRANSFER_PKEY,
-        sender=UNIT_TRANSFER_SENDER,
-        compute_locksroot=False,
-        allow_invalid=False,
+        channel_state: NettingChannelState = EMPTY,
+        amount: typing.TokenAmount = EMPTY,
+        initiator: typing.InitiatorAddress = EMPTY,
+        target: typing.TargetAddress = EMPTY,
+        expiration: typing.BlockExpiration = EMPTY,
+        secret: typing.Secret = EMPTY,
+        identifier: typing.PaymentID = EMPTY,
+        nonce: typing.Nonce = EMPTY,
+        transferred_amount: typing.TokenAmount = EMPTY,
```

Comment:
I also asked this question in Rocketchat but I am not sure where to use `typing.TokenAmount` and where to use `typing.PaymentAmount`. They seem to be used interchangably at the moment and I would rather have only one.

Reply:
Then perhaps let's stick to TokenAmount everywhere as mypy has complained in a few places about it when I used it on mediator.py